plugins/org.eclipse.sirius.tests.swtbot.support/src/org/eclipse/sirius/tests/swtbot/support/api/AbstractSiriusSwtBotGefTestCase.java

```diff
@@ -1067,6 +1069,48 @@ protected void movePinnedElements() {
         SWTBotUtils.waitAllUiEvents();
     }
 
+    /**
+     * Returns the bot for the DropDownButton, which contains all paste actions.
+     */
+    protected SWTBotToolbarDropDownButton getPasteMenu() {
+        String[] tooltips = { Messages.AdaptivePasteStyleAction_toolTipText, Messages.PasteFormatAction_toolTipText_diagram, Messages.PasteFormatAction_toolTipText_diagramElements,
+                Messages.PasteLayoutAction_toolTipText_diagram, Messages.PasteLayoutAction_toolTipText_diagramElements, Messages.PasteStyleAction_toolTipText_diagram,
+                Messages.PasteStyleAction_toolTipText_diagramElements, Messages.PasteImageAction_toolTipText,
+        };
+        List<Matcher<? extends ToolItem>> tooltipMatchers = new ArrayList<>();
+        Arrays.stream(tooltips).forEach(tooltip -> tooltipMatchers.add(WidgetMatcherFactory.withTooltip(tooltip)));
+        Matcher<ToolItem> tooltipMatcher = WidgetMatcherFactory.anyOf(tooltipMatchers);
+        Matcher<ToolItem> typeMatcher = WidgetMatcherFactory.widgetOfType(ToolItem.class);
+        Matcher<ToolItem> styleMatcher = WidgetMatcherFactory.withStyle(SWT.DROP_DOWN, "SWT.DROP_DOWN");
+        Matcher<ToolItem> matcher = WidgetMatcherFactory.allOf(List.of(typeMatcher, tooltipMatcher, styleMatcher));
+        return new SWTBotToolbarDropDownButton(editor.bot().widget(matcher), matcher);
+    }
+
+    /**
+     * This method returns whether an item of a DropDownButton is enabled or not.
+     */
+    protected boolean dropDownButtonItemIsEnable(SWTBotToolbarDropDownButton dropDownButton, String itemText) {
+        if (dropDownButton.isEnabled()) {
+            SWTBotMenu item = dropDownButton.menuItem(itemText);
+            boolean isEnable = item.isEnabled();
+            dropDownButton.pressShortcut(Keystrokes.ESC);
+            return isEnable;
```

Comment:
why is it necessary? because you did not open the menu so no reason to close it with Esc?

Reply:
Yes, the menu is open. I have to close it.